test.js

```diff
@@ -1001,7 +1002,8 @@ experiment('ignore request logs for paths in ignorePaths', () => {
       resolver = resolve
     })
     const stream = sink((data) => {
-      expect(data.req.url).to.not.equal('/ignored')
+      const url = new URL(data.req.url)
+      expect(url.pathname).to.equal('/')
```

Comment:
I'm wondering why these tests where passing before. Did you get a clue on why?

Reply:
`data.req.url` was `undefined`, so it passed because it wasn't `/ignored`.

Reply:
Sorry, not undefined, but it was a full URL (http://colins-macbook-pro.local:0/)